source/_layouts/default.html

```diff
@@ -85,9 +83,8 @@
             <span class="text-muted">&copy; {{ "now"|date("Y") }} {{ site.title }}</span>
         </footer>
 
-        <script src="{{ site.url }}/components/jquery/jquery.min.js"></script>
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
-        <script src="{{ site.url }}/components/bootstrap/js/bootstrap.min.js"></script>
+        <script src="{{ site.url }}/build/runtime.js"></script>
+        <script src="{{ site.url }}/build/app.js"></script>
```

Comment:
why do we need more than one js file, if they are both included at the bottom? usually, when you get more than 1, the idea is to load the minimum required thing in the head, then have content so it can already be rendered, and then load extra js at the end, for fastest page impression. if both are loaded at the end, why not combine them into one?

Reply:
I think the idea is that Webpack includes common JS dependencies in the runtime.js file, so that if you have JS files for specific pages (e.g. a checkout.js that's only required in a checkout page), you don't have to duplicate those dependencies. https://symfony.com/blog/encore-0-21-0-webpack-4-support-copy-files-support-webpackencorebundle#new-runtime-js-file

The intro docs say 'you probably want this, unless you're building a single-page app' but in this case, it does seem to be unnecessary.

Reply:
oh, i remember that from back in the days when i did front end things. religious question... 

i agree with you: for a generated static site, single js looks fine. people who have heavy js stuff on only some pages will know how to do their thing (and chances are you need a backend application and then don't use sculpin anyways). do you change that?

Reply:
Yes, I'm going to change it. :)